package.json

```diff
@@ -61,6 +63,21 @@
   ],
   "devDependencies": {
     "cross-env": "^5.2.0",
-    "gh-pages": "^2.0.1"
-  }
+    "gh-pages": "^2.0.1",
+    "speculate": "^2.0.0"
+  },
+  "spec": {
+    "requires": [
+      "cloud-httpd24-ssl-services-devs-staff"
+    ],
+    "executable": [
+      "./bake-scripts/*"
+    ],
+    "post": [
+      "mkdir -p /etc/bake-scripts/digital-paper-edit-client",
+      "cp -rf /usr/lib/digital-paper-edit-client/bake-scripts/* /etc/bake-scripts/digital-paper-edit-client/",
+      "./scripts/environment-config.py"
```

Comment:
not entirely sure you do need this, it should just get called post installation?